add kickstart reminder email (#22)
srobo/volunteer-emails: SR2020/2019-10-19-kickstart-reminder.md

> # Student Robotics Kickstart Info

Reviewer: Can haz header info. A la https://github.com/srobo/volunteer-emails/blob/master/SR2020/2019-09-24-kit-sorting-day.md

Hi all,

Kickstart is happening in exactly 1 week - and we could really do with your help to ensure it runs smoothly!

> It'll run from 9am to 5pm (or whenever teams are finished)

Reviewer: ```suggestion
It'll run from 10am to 5pm (or whenever teams are finished)
```

> **If you'd like to help out and haven't signed up already, [fill out the Kickstart sign up form](https://forms.gle/1CPgReTxhCGn6rcx6)!**

Reviewer: ```suggestion
If you'd like to help out, and haven't signed up already, please [fill out the Kickstart sign up form](https://forms.gle/1CPgReTxhCGn6rcx6)!
```

We'll be hosting it simultaneously in 3 separate locations:

 - [Southampton](https://studentrobotics.org/events/sr2020/southampton-kickstart/)
 - [Central London](https://studentrobotics.org/events/sr2020/london-kickstart/)
 - [Cambridge](https://studentrobotics.org/events/sr2020/cambridge-kickstart/)

Also here are the details you need to know to help out:

> Kickstart is scheduled to start at 10, so **arrive at 9 to help with preparations**.

Reviewer: I think this should be worded slightly. If people turn up for 10 that's totally fine. Turning up at 9 is greatly appreciated, rather than expected.

The exact details on how to to go for each kickstart is as follows:

### University of Southampton:
    Building 53
    University Rd
    Southampton
    SO17 1BJ

> Come to Building 53 if you're on time, Meet at the seminar room on the top floor. The building is ID-card access only, so you'll need to call us to be let in. If you arrive past 9:30ish or if there's nobody there, follow the directions on the website.

Reviewer: ```suggestion
Come to Building 53 if you're arriving for 9am - Meet at the seminar room on the top floor (`53/4025`). The building is ID-card access only, so you'll need to send a message in [`#southampton`](https://studentrobotics.slack.com/messages/southampton) to be let in. If you arrive past 9:30ish or if there's nobody there, follow the directions on the website.
```

### London
    Thread
    1 Alie Street
    Whitechapel
    London
    E1 8DE

When arriving at reception, ask for Thread.

### Cambridge
    William Gates Building
    15 JJ Thomson Ave
    Cambridge
    CB3 0FD

> When arriving, go to Lecture Theatre 1 (to the right when entering through reception).

Reviewer: Some kind of closing statement would be quite nice, primarily thanking people for committing their time.

---
to: All Volunteers
subject: SR2020 Kickstart Info & Call for volunteers
---

Hi all,

> Kickstart is happening in exactly 1 week - and we could really do with your help to ensure it runs smoothly!

Reviewer: Nit: _Technically_ It's in less than a week.

```suggestion
Kickstart is less than 1 week away - and we could really do with your help to ensure it runs smoothly!
```

It'll run from 10am to 5pm (or whenever teams are finished)

If you'd like to help out and haven't signed up already, [fill out the Kickstart sign up form](https://forms.gle/1CPgReTxhCGn6rcx6) so we know who's coming!

We'll be hosting it simultaneously in 3 separate locations:

 - [Southampton](https://studentrobotics.org/events/sr2020/southampton-kickstart/)
 - [Central London](https://studentrobotics.org/events/sr2020/london-kickstart/)
 - [Cambridge](https://studentrobotics.org/events/sr2020/cambridge-kickstart/)

Also here are the details you need to know to help out:

> Kickstart is scheduled to start at 10, but you can arrive at 9am to help with preparations if convenient.

Reviewer: ```suggestion
Kickstart is scheduled to start at 10, but setup starts at 9am if you're around to help out.
```

> Also if you already have an SR t-shirt from a previous year (preferably from last year), please bring it to ensure you have one!

Reviewer: Worth mentioning we'll have some spares.

```suggestion
Also if you already have an SR t-shirt from a previous year (preferably from last year), please bring it to ensure you have one! We'll have some spares if you don't already have one.
```
